applet/src/main/java/applet/EmeraldApplet.java

```diff
@@ -22,15 +26,18 @@ Team Emerald (in alphabetical order):
      * TODO `javacard.framework.OwnerPIN` is not suitable as we need to read PIN value for J-PAKE
      */
     private final byte[] pin; // TODO store pin securely
```

Comment:
I wonder if we ever get our hands on this.

Reply:
I could at least encrypt that with some static key...